src/diffusers/pipelines/tune_a_video/__init__.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Union
+
+import numpy as np
+import torch
+
+from ...utils import (
+    BaseOutput,
+    OptionalDependencyNotAvailable,
+    is_flax_available,
+    is_k_diffusion_available,
+    is_k_diffusion_version,
+    is_onnx_available,
+    is_torch_available,
+    is_transformers_available,
+    is_transformers_version,
+)
+
+
+@dataclass
+class TuneAVideoPipelineOutput(BaseOutput):
```

Comment:
If we do this, maybe it makes sense for us to expose a utility function that can take a tensor and create a video out of it. 

@patrickvonplaten @williamberman WDYT?

Reply:
yeah that makes sense. But will we not be introducing a dependency on ffmpeg or someother lib of that sort ?

Reply:
Yes, we need to discuss this bit. 

Personally, I am fine if we just return the video tensors (PyTorch or NumPy). Since we already cover how we can create a video out of the tensors in the docs, I think it should be fine. 

We just make it clear in the docs that the users should install a certain library to be able to serialize the tensors as videos.

Reply:
I think the new release has a `export_to_video` method in diffusers.utils.